### ISSUE TITLE
Example of ARIA 1.1 combobox without autocomplete

### ISSUE THREAD

**Issue by carmacleod** [w3c/aria-practices#990]
Currently, all of the ARIA 1.1 combobox examples have autocomplete.
None of them are announced a "combobox" by any screen reader that I tried (only "edit, has autocomplete).
Would be useful to have an example that does not have autocomplete, for testing regular ARIA 1.1 combobox in screen readers.

**Comment by charmarkk** [comment]
Are you look for an equivalent for the [legacy 1.0 example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/combobox/aria1.0pattern/combobox-autocomplete-none.html), Carolyn? That is to say, are you looking for a 1.1 example with `autcomplete=none`? We just need some clarification. :)

**Comment by carmacleod** [comment]
Yes, please! Except please don't use the `autocomplete` attribute at all. Thanks! :)

**Comment by mcking65** [comment]
We now have an example with autocomplete=none in the APG 1.2 draft. And, per the spec, autocomplete is required. So we will not add an example that does not have the aria-autocomplete attribute.

Although, Aaron Leventhal has argued that autocomplete should not be required if the combobox is not editable, and I agree that could be a good improvement. However, I do not think we should propose further changes at this time unless they are necessary to get 1.2 done.

Closing as this has been overcome by events.
